app/controllers/group_members_controller.rb

```diff
@@ -0,0 +1,72 @@
+class GroupMembersController < ApplicationController
+  before_action :authenticate_user!
+
+  def index
+    group = Group.find(params[:group_id])
+
+    res = current_user &&
+      (group.is_admin?(current_user) || group.is_mod?(current_user))
```

Comment:
We're using `current_member.admin?` in some places and `group.is_admin?(current_user)` in others. Should make this consistent.